### ISSUE TITLE
Crash when interact with hashtags

### ISSUE THREAD

**Issue by bronzeboy** [alexal1/Insomniac#158]
Seems can't handle any hashtag, it seems to have problem with scroll to end

**Comment by alexal1** [comment]
Could you please attach Crash.zip folder?
It may happen because different accounts can have slightly different UI and crash info will help to fix it.

**Comment by bronzeboy** [comment]
> Could you please attach Crash.zip folder?
> It may happen because different accounts can have slightly different UI and crash info will help to fix it.

It can't scroll, that's the function that crash. I'll report here the error but I think it would be great if the report autogenerated would be anonymous with personal information taken out!

**Comment by MelBourbon** [comment]
Maybe this will help. I think I have the same issue on an "old" device running with Android 5
```
Traceback (most recent call last):
  File "<string>", line 174, in scroll
  File "/usr/local/lib/python3.7/dist-packages/uiautomator2/_selector.py", line 575, in __call__
    max_swipes, steps)
  File "/usr/local/lib/python3.7/dist-packages/uiautomator2/__init__.py", line 436, in __call__
    http_timeout)
  File "/usr/local/lib/python3.7/dist-packages/uiautomator2/__init__.py", line 442, in _jsonrpc_retry_call
    return self._jsonrpc_call(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/uiautomator2/__init__.py", line 514, in _jsonrpc_call
    raise err
uiautomator2.exceptions.UiObjectNotFoundError: -32001 Jsonrpc error: <android.support.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=androidx.recyclerview.widget.RecyclerView, RESOURCE_ID=android:id/list], method: scrollToEnd

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/insomniac/__init__.py", line 418, in wrapper
    func(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/insomniac/__init__.py", line 227, in job
    on_interaction)
  File "<string>", line 114, in handle_hashtag
  File "<string>", line 176, in scroll
insomniac.device_facade.DeviceFacade.JsonRpcError: -32001 Jsonrpc error: <android.support.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=androidx.recyclerview.widget.RecyclerView, RESOURCE_ID=android:id/list], method: scrollToEnd
```

**Comment by Fadude** [comment]
Fixed on 3.5.0
